README.md

```diff
@@ -142,7 +142,9 @@ Using a debugger will allow you to pause and inspect the server code as it is be
 
 If you do not want to use Docker at all, you can also [install the database and search dependencies on your machine](./DEPENDENCIES_MANUAL_INSTALL.md)
 
+## Watch files and live reload with Webpack
 
+ Here is the link to [Watch files and live reload with Webpack](./NODEJS_SETUP.md)
```

Comment:
You could be more descriptive. What if I don't know what live reloading or webpack are?